README.md

```diff
@@ -3,29 +3,29 @@
 [![Build Status](https://travis-ci.org/pingcap/tidb-insight.svg?branch=master)](https://travis-ci.org/pingcap/tidb-insight)
 [![Go Report Card](https://goreportcard.com/badge/github.com/pingcap/tidb-insight)](https://goreportcard.com/report/github.com/pingcap/tidb-insight)
 
-This is a set of tools and/or scripts to collect infomation and metrics from all nodes of a TiDB cluster, the data are collected and archived to a tarball, so that one can copy the data to anywhere convenient for further invesgating.
+This is a set of tools and/or scripts to collect information and metrics from all nodes of a TiDB cluster. The data is collected and archived to a tarball so that one can copy the data to anywhere convenient for further investigation.
```

Comment:
-> TiDB Insight is a set of tools and/or scripts to collect and analyze information and metrics from all nodes of a TiDB cluster. The data is collected and archived to a tarball so that one can copy the data anywhere for further investigation.